utils/checksums.jl

```diff
@@ -2,12 +2,12 @@
 
 using Downloads, SHA
 
-const cliversion = v"0.7.5"
-const base_url = "https://github.com/codecov/codecov-cli/releases/download/v0.7.5/"
+const cliversion = v"9.0.4"
```

Comment:
Oh nvm, something weird happening with upstream versions apparently.

Reply:
> Due to an issue with versioning, the releases have skipped to v9.0.4 from v0.9.4. In order to prevent downstream issues with users who may have already upgraded, we will continue to release from 9.0.4.

https://github.com/codecov/codecov-cli/releases/tag/v9.0.4